lib/Yield/CMakeLists.txt

```diff
@@ -85,3 +81,17 @@ target_tuple_tree_generator(
   "${INTERNAL_ASSEMBLY_HEADERS}/RelationDescription.h"
   "${INTERNAL_ASSEMBLY_HEADERS}/RelationTarget.h"
   "${INTERNAL_ASSEMBLY_HEADERS}/RelationType.h")
+
+revng_add_analyses_library_internal(
+  revngYieldPipes
+  SHARED
+  Pipes/AssemblyPipes.cpp
+  Pipes/CallGraphPipes.cpp
+  Pipes/CFGPipes.cpp)
+
+target_link_libraries(
+  revngYieldPipes
+  revngYield
+  revngFunctionIsolation
```

Comment:
Yep, as far as I see now, `CrossRelations` only depends on `yield::Graph` (outside of the `model` and `efa` dependencies) so this should not be a problem.

My only concern is what I should rename the `yield::Graph` to then.

Reply:
Its core property seems to a graph where nodes have a position in the plain. `LaidOutGraph`?

Reply:
I guess that would work, but I'm not excited. Since the graph does not get created in `LaidOff` state, you need to run a `Layouter` separately. This could lead to confusion.

Also, where's the boundary we're cutting stuff on? I feel like wherever the graph goes, the layouter should go too, but I guess that's it.

Should Graph + layouters + relations should be a single library? Or two distinct ones (as in, graph stuff has nothing to do with the relations)?

Reply:
I feel like we should decouple the emission of graphs from the laying out.

Something like:
```c++
GenericGraph emitCrossRelations();
GenericGraph emitCFG();
yield::Graph layOut(GenericGraph);
```
Is there something preventing us to go this route?

Reply:
This would make the layout function super-specialized. But maybe that's not such a bad thing, tbh.

But I'd still prefer for both graphs to be explicitly named so it's clear what they are (of course they are both `GenericGraph`s).

Reply:
Can you link the pad?

Reply:
I linked it in `~asm-and-graphs` header, but sure, here it is: https://pad.rev.ng/s/S3VedMpAo